samples/https/README.md

```diff
@@ -0,0 +1,23 @@
+# HTTPS service
+
+This sample consists of an NGINX service which can encrypt traffic using HTTPS.
```

Comment:
I would remove which can encrypt traffic and leave just: _This sample defines an HTTPS service backed by NGINX._

Reply:
fixed. Thanks.

Reply:
Doesn't seem fixed?